docs/REST API/rest-api.md

```diff
@@ -8550,11 +8552,12 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ### Add software
 
```

Comment:
### Dev note

Error message returned in the API when the user attempts to add a package and Fleet can't extract the version.

Here's what this error message will look like in the UI: https://www.figma.com/design/4pfUOYy7IyMIrjMH2fuCdU/%2319551-Automatically-install-software-and-scope-software-with-labels?node-id=5568-4943&t=ZBrK0Ypx0LwOLF5n-1